shipyard_blueprints/postgresql/shipyard_postgresql/errors/exceptions.py

```diff
@@ -0,0 +1,23 @@
+from logging import error
+from shipyard_templates import Database, ExitCodeException
+
+EXIT_CODE_UPLOAD_ERROR = 101
+EXIT_CODE_CHUNK_DOWNLOAD_ERROR = 102
+
+
+class InvalidCredentials(ExitCodeException):
+    def __init__(self, error_message: Exception):
+        self.message = f"Error in connecting to Postgres. Message from the server reads: {error_message}"
+        self.exit_code = Database.EXIT_CODE_INVALID_CREDENTIALS
+
+
+class PostgresUploadError(ExitCodeException):
```

Comment:
Should use the database class otherwise the template abstract class exceptions don't really add value.